src/patternfly/demos/Card/card-view-demo-template-gallery.hbs

```diff
@@ -180,4 +180,17 @@
         Expose REST services and their APIs using Swagger specification.
       {{/card-body}}
     {{/card}}
+    {{#> card card--id="card-11" card--modifier="pf-m-compact pf-l-bullseye"}}
+      {{#> empty-state empty-state--modifier="pf-m-xs"}}
+        <i class="fas fa-plus-circle pf-c-empty-state__icon"></i>
+        {{#> title titleType="h1" title--modifier="pf-m-md"}}
```

Comment:
It probably makes more sense here to mimic the title in the other cards. So you can use `{{#> card-title}}<p> To get started...</p>{{/card-title}}`